FHIR-Converter/README.md

```diff
@@ -92,11 +95,11 @@ If a converter workspace is already opened and you want to switch to another con
 
 ![open-switch-workspace](assets/open-switch-workspace.gif)
 
-### 6. Convert the HL7v2 messages into FHIR bundles
+### 6. Convert the HL7v2/C-CDA messages into FHIR bundles
 
 In a converter workspace, template files and data files are shown in the explorer view. To start template editing, select a template file and a data file.
 
-To select a file as the root template file, right click on the file having extension `.liquid` in the explorer view and select the menu item `FHIR Converter: Select as template (*.liquid)`. Similarly, to select a data file, right click on the data file having extension `.hl7` and select the menu item `FHIR Converter: Select as data file (*.hl7)`.
+To select a file as the root template file, right click on the file having extension `.liquid` in the explorer view and select the menu item `FHIR Converter: Select as template (*.liquid)`. Similarly, to select a data file, right click on the data file having extension `.hl7` or `.ccda` and select the menu item `FHIR Converter: Select as data file (*.hl7 or *.ccda)`.
```

Comment:
The C-CDA command-line tools accepts both `.ccda` and `.xml` as input data. Do we need to support `.xml` in the extension?

Reply:
Sure, I can update it.